contents/docs/product-analytics/troubleshooting.mdx

```diff
@@ -63,49 +63,51 @@ posthog = PostHog::Client.new({
 
 </MultiLanguage>
 
-3. Make sure that the SDK sends a couple of events.
-4. See if the incoming events arrive as expected.
+3. Send a few events from your app.
+
+If no events are *not* visible in webhook.site, that means you have misconfigured your PostHog library. Consult our [installation guide](/docs/product-analytics/installation) for how to initialize PostHog.
+
+If events are indeed visible in webhook.site, these means that you have configured your PostHog correctly.
```

Comment:
Not necessarily, the events could still be in poor format and them having configured them badly. But we can now assume that if we can reach webhook.site, then we can also reach app.posthog.com with the same payload as we sent to webhook.site

Reply:
> Not necessarily, the events could still be in poor format and them having configured them badly.

Is the correct term here then `you have initialized posthog correctly` instead of "you have configured posthog correctly` ?

Otherwise Im not sure why we check webhook.site, if the title of this section is "you have configured posthog incorrectly" – and we should suggest ways to check you have configured posthog correctly

Reply:
sure ... I'm not sure where the questions around properties are missing / events aren't being flushed belong to ... happy for you to make that call